airgun/entities/host.py

```diff
@@ -253,7 +253,9 @@ def get_webconsole_content(self, entity_name):
         view.validations.assert_no_errors()
         # switch to the last opened tab,
         self.browser.switch_to_window(self.browser.window_handles[-1])
-        self.browser.wait_for_element(locator='//div[@id="index-brand"]', exception=True)
+        self.browser.wait_for_element(
+            locator='//div[@id="index-brand"]', exception=True, timeout=10
```

Comment:
I believe the timeout is very less, is this expected and intentional ?

Reply:
Hi, I can't see how this is related to my change

Reply:
@swadeley your commits are off, looks like you might have rebase/amended too far back. That's why this is showing up in your PR.

Reply:
I can rebase again?

Reply:
@jyejare @mshriver I deleted that commit and mine, rebased, c-picked my commit, should be OK now. Now idea how it happened, honest.